### DISCUSSION TITLE
Server not starting anymore

### DISCUSSION THREAD

**Discussion by wuast94** [minio/minio#18377] [Q&A] [answered]
## Expected Behavior
Server starting normally

## Current Behavior
Server not starting anymore

## Possible Solution
More Logging would be helpful at least

## Steps to Reproduce (for bugs)

Start the Docker Container

## Context

My Minio Server is running with docker, had this the last time but back then i just deleted everything and startet over again.
The Server is running for a few month now without a problem, no its broken again.
S3 endpoint not working and enter the admin url i get a 404.

Intresting, if i stop the container, rename the mapped folder and start the container again it starts without problem. so i would say there is a config broken or something like that. 

## Your Environment

* Version used (`minio --version`):
```
minio version RELEASE.2023-11-01T18-37-25Z (commit-id=4b4a98d5e59354870325ad19703fba03d1b104c2)
Runtime: go1.21.3 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2023 MinIO, Inc.
```

* Server setup and configuration:
```
docker --version
Docker version 24.0.7, build afdd53b4e3
```
* Logs:
```
MinIO Object Storage Server
Copyright: 2015-2023 MinIO, Inc.
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Version: RELEASE.2023-11-01T18-37-25Z (go1.21.3 linux/amd64)
Status:         1 Online, 0 Offline. 
S3-API: https://s3.wuast24.de 
Console: https://s3admin.wuast24.de/ 
Documentation: https://min.io/docs/minio/linux/index.html
Warning: The standard parity is set to 0. This can lead to data loss.
```

* Operating System and version (`uname -a`): 
`Linux server 6.5.9-zen2-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Thu, 26 Oct 2023 00:51:53 +0000 x86_64 GNU/Linux`

**Comment by klauspost** [comment]
The requests are not reaching MinIO. The 404 response is not something returned by MinIO.

**Reply by wuast94** [reply]
Yes the 404 ist coming from my traefik Proxy. But the internal healthcheck isnt working eather. And as i Said, Same Setup Just fresh because of moving the mapped folder works, then everything Comes Up again. So it must bei some minio issue

**Reply by klauspost** [reply, marked as answer]
Could it be because of this: https://github.com/minio/minio/issues/18373

**Reply by wuast94** [reply]
changed the Healthcheck mentioned there and now it starts again, makes no sense for me but thanks for the quick response 😊
